pkg/actuators/machine/reconciler.go

```diff
@@ -73,14 +74,17 @@ func (r *Reconciler) create() error {
 	}
 
 	if err = r.updateLoadBalancers(instance); err != nil {
+		metrics.RegisterFailedInstanceCreate(&metrics.MachineLabels{
+			Name:      r.machine.Name,
+			Namespace: r.machine.Namespace,
+			Reason:    err.Error(),
+			Provider:  metrics.AWSProvider,
+		})
 		return fmt.Errorf("failed to updated update load balancers: %w", err)
 	}
 
 	klog.Infof("Created Machine %v", r.machine.Name)
-
-	if err = r.setProviderID(instance); err != nil {
-		return fmt.Errorf("failed to update machine object with providerID: %w", err)
-	}
+	r.setProviderID(instance)
```

Comment:
why is this ignoring this error now?

Reply:
The method always returns nil as an error.

Reply:
how is this related to this commit?

Reply:
It is not, but I didn't like it 😅